src/utils.ts

```diff
@@ -373,7 +372,7 @@ export interface LaunchOptions {
     /** Whether to run the browser in headless mode. Defaults to `false`.
      * Note: On Linux, passing `headless='virtual'` will use Xvfb.
      */
-    headless?: boolean;
+    headless?: boolean | 'virtual';
```

Comment:
this was matching the [original Python project's API](https://github.com/daijro/camoufox/blob/95cc4489d025471368412b035743bb59855b160e/pythonlib/camoufox/utils.py#L357).